### PR TITLE
More revising

### PR DESCRIPTION
**Pull request by dabrahams**
Dimitri, please make sure my deletions seem OK to you.  It seemed like those passages weren't adding anything useful for this audience.

### CODE REVIEW

File: pages/language-tour.md [hunk 1]
```diff
@@ -80,41 +74,52 @@ To run this program:
 - Run the command `valc Hello.val Greet.val -o hello`
 - Run the command `./hello`
 
-*Alternatively, you may put both source files in a folder, say `Sources`, and compile the program with `valc Sources -o hello`.*
+*Alternatively, you may put both source files in a subdirectory, say `Sources/`, and compile the program with `valc Sources -o hello`.*
 
 Note that `greet` need not to be `public` to be visible from another file in the module.
 All entities declared at the top level of a file are visible everywhere in a module, but not beyond that module's boundary.
+{% comment %}
+Do we need to say, "unless marked private?"
```
Comment (kyouko-taiga) [inline pages/language-tour.md:82]: You mean `public`?
We don't have a private accessor at the moment.

File: pages/language-tour.md [hunk 2]
```diff
@@ -80,41 +74,52 @@ To run this program:
...
+{% endcomment %}
 
 ### Bundling modules
 
-The simplest way to work with multiple modules is to gather source files into different folders.
-For example, let's move `greet` in a different module, using the following arborescence:
+The simplest way to work with multiple modules is to gather source files into different subdirectories.
+For example, let's move `greet` in a different module, using the following directory structure:
+{% comment %}
+I love the word “arborescence!”  But nobody will know what it means.
```
Comment (kyouko-taiga) [inline pages/language-tour.md:90]: French speakers will ;)

File: pages/language-tour.md [hunk 3]
```diff
@@ -278,35 +231,31 @@ public fun main() {
...
+  var weight: Double = 1.0
   weight = 2.3
   print(weight) // 2.3
 }
 ```
+
+The type of an expression can be retrieved, without evaluating the expression, using `type(of:)`:
 {% comment %}
-The ability to use (e.g.) an integer literal to initialize a double is IIUC one of the great sources of expression too complex errors in Swift.
+I understand, precedent from Swift, but… what purpose does `of:` serve here?!
```
Comment (kyouko-taiga) [inline pages/language-tour.md:246]: I was about to change and then it felt wrong when I saw `type(weight)` in the code. I guess I'm used to `typeof` javascript, so the label seems natural to me.

**Review by kyouko-taiga — APPROVED** [review]
LGTM

File: pages/language-tour.md (hunk 1 above)
Comment (dabrahams) [reply pages/language-tour.md:82]: Oh.  KISS, then.

File: pages/language-tour.md (hunk 3 above)
Comment (dabrahams) [reply pages/language-tour.md:246]: Yeah, other languages too.

**Pull request merged**